docs/products/mysql/concepts/mysql-replication.rst

```diff
@@ -0,0 +1,31 @@
+MySQL replication
+=================
+
+.. _replication-overview:
+
+Overview
+--------
+
+MySQL replication is always based on replicating logical changes. This means that the replication protocol may contain an actual statement that the target server should apply or it may have an entry saying **update row with these old attributes to have these new attributes**. These are called statement and row formats.
+
+The statement format is more compact but cannot represent all changes because some statements would yield different results if executed as is on different servers. The row format statement can represent all changes, and it allows using tools like Debezium since the binary log contains full details of all changes in itself. For these reasons Aiven uses row format by default. 
```

Comment:
I'm not sure I understand what is the difference between a statement and row format. Maybe an example would showcase better?

Reply:
Added an external link with more details